pkg/apis/intevents/v1alpha1/brokercell_validation.go

```diff
@@ -18,12 +18,119 @@ package v1alpha1
 
 import (
 	"context"
+	"fmt"
 
+	resourceutil "github.com/google/knative-gcp/pkg/utils/resource"
+	"k8s.io/apimachinery/pkg/api/resource"
 	"knative.dev/pkg/apis"
 )
 
+const (
+	avgMemoryUsageOutOfRangeErrorDetail  string = "AvgMemoryUsage should not exceed the memory limit"
+	resourceRequestOutOfRangeErrorDetail string = "Resource request should not exceed the resource limit"
+	unexpectedQuantityFormatErrorDetail  string = "The quantity is specified in an unexpected format"
+)
```

Comment:
nit: IMO it's pretty rare in Go to define constants like these. Usually for strings that are only used once, my preference is to make them inline.

Reply:
Some of these are used more than once (for both CPU and memory) and all of them are referred to from unit tests. Is it better to duplicate strings or any other good practice for that?